specs/waku/v2/waku-filter.md

```diff
@@ -66,8 +66,19 @@ message FilterRequest {
 message MessagePush {
   repeated WakuMessage messages = 1;
 }
+
+message FilterRPC {
```

Comment:
Request id?

Reply:
Added UUI

Reply:
I wonder if this shouldn't be RequestId, uuid seems a bot generic

Reply:
i.e. *what* it is is a RequestId. UUID is a type. So conceptually it is something like:

```
uuid RequestId = 1
...
~ uuid = string # basically type alias
```

In more enlightened languages this would be type checked to have a more restricted type, e.g. with always have 28 characters or whatever it is. This would mean using something like "1" would fail (as it should)

Reply:
We can skip the uuid type alias though, and do validation at impl layer.

Here's how google defines it: https://developers.google.com/search/apis/indexing-api/v3/reference/indexing/rpc/google.api

Reply:
I changed the field name, I wouldn't say that `1` is invalid. The rule to me should just be that it is a unique.

Reply:
if someone uses 1 as a request you are bound to run into bugs as requests gets mangled...